app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+
+  <string name="scan_request_camera_access">Povolit přístup ke kameře</string>
+  <string name="scan_instructions">Umístěte QR kód platebního požadavku dovnitř obdélníčku, aby mohl být přečten</string>
+  <string name="scan_state_reading">Čtení platebního požadavku</string>
+
+  <string name="scan_error_default">Toto není platná žádost o platbu.\n\nProsím, zkuste to znovu.</string>
+  <string name="scan_error_expired">Platba vypršela.</string>
+  <string name="scan_error_pay_to_self">Nemůžete platit sám sobě.</string>
+  <string name="scan_error_invalid_chain">Tento požadavek nepoužívá stejný blockchain jako vaše peněženka.</string>
+  <string name="scan_error_lnurl_unsupported">Tato LNURL není zatím podporována!</string>
+
+  <string name="scan_amountless_legacy_title">Varování</string>
+  <string name="scan_amountless_legacy_message"><![CDATA[
+    Tento požadavek neobsahuje množství. To může být nebezpečné: zlomyslné uzly se mohou pokusit ukrást vaší platbu. Z hlediska bezpečnosti bystě měli <b>požádat placeného, aby stanovil množství</b> v platebním požadavku.
+    <br /><br />
+    Jste si jistí, že chcete zaplatit tento požadavek?
+  ]]></string>
+  <string name="scan_amountless_legacy_confirm_button">Potvrdit</string>
+  <string name="scan_amountless_legacy_cancel_button">Zrušit</string>
+
+  <!-- //////////////// utilities strings //////////////// -->
+
+  <!-- #1: sign / #2: integer part / #3: decimal part / #4: unit -->
+  <string name="utils_converted_amount">≈ %1$s</string>
+  <string name="utils_pretty_amount"><![CDATA[%1$s%2$s<small>%3$s<small>]]></string>
+  <string name="utils_pretty_amount_with_unit"><![CDATA[%1$s%2$s<small>%3$s<small> %4$s]]></string>
+
+  <string name="utils_ok">Ok</string>
+  <string name="utils_proceed">Pokračovat</string>
+  <string name="utils_cancel">Zrušit</string>
+  <string name="utils_date_just_now">Právě teď</string>
+  <string name="utils_unknown">N/A</string>
+  <string name="utils_copied">Zkopírováno do schránky!</string>
+  <string name="btn_back">Zpět</string>
+  <string name="btn_ok">OK</string>
+  <string name="btn_confirm">Potvrdit</string>
+  <string name="btn_cancel">Zrušit</string>
+  <string name="btn_close">Zavřít</string>
+
+  <!-- //////////////// settings //////////////// -->
+
+  <string name="settings_general_title">Obecné</string>
+  <string name="settings_display_prefs">Zobrazení</string>
+  <string name="settings_security_title">Bezpečnost</string>
+  <string name="settings_display_seed">Obnovovací fráze</string>
+  <string name="settings_seed_security">Nastavení přístupu</string>
+  <string name="settings_advanced_title">Pokročilé</string>
+  <string name="settings_fees">Nastavení poplatků</string>
+  <string name="settings_list_channels">Seznam kanálů</string>
+  <string name="settings_logs">Logy</string>
+  <string name="settings_electrum">Electrum server</string>
+  <string name="settings_mutual_close">Vynucené uzavření kanálů</string>
```

Comment:
> Vynucené uzavření kanálů

This button points to the mutual close page, where the user can cooperatively close his channels and send funds to an on-chain address. A force-close is very different. 

I do not speak Czech, however Google translates this string as `Forced channel closure`, which would not be correct. Can you confirm that the translation is consistent?

Reply:
If this is the button title then I'll fix that. Thanks

Reply:
You're right, different button. Thanks.